common/test/acceptance/pages/studio/html_component_editor.py

```diff
@@ -58,3 +61,115 @@ def set_raw_content(self, content):
         type_in_codemirror(self, 0, content)
 
         self.q(css='.mce-foot .mce-primary').click()
+
+    def open_settings_tab(self):
+        """
+        Clicks settings button on the modal
+        """
+        self._click_button(self.settings_tab)
+
+    def save_settings(self):
+        """
+        Click save button on the modal
+        """
+        self._click_button(self.save_settings_button)
+
+    def open_link_plugin(self):
+        """
+        Opens up the link plugin on editor
+        """
+        self.q(css='[aria-label="Insert/edit link"]').click()
+        self.wait_for_element_visibility('.mce-window-head', 'Window header present')
+
+    def save_static_link(self, static_link):
+        """
+        Adds static link inside the link plugin
+        """
+        self.q(css='.mce-combobox .mce-textbox').fill(static_link)
+        self.q(css='.mce-btn.mce-primary').click()
+
+    @property
+    def href(self):
+        """
+        Gets the href from the editor
+        """
+        return self.q(css="#tinymce>p>a").attrs('href')[0]
+
+    def switch_to_iframe(self):
+        """
+        Switches to the editor iframe
+        """
+        self.browser.switch_to_frame(self.browser.find_element_by_tag_name('iframe'))
+
+    @property
+    def url_from_the_link_plugin(self):
+        """
+        Clicks the already set link from the editor and then returns the URL from the link plugin
+        """
+        # self.q(css='#tinymce>p>a').first.click()
+        # self.browser.switch_to_default_content()
+        self.open_link_plugin()
+        return self.browser.execute_script('return $(".mce-textbox").val();')
+
+    def get_default_settings(self):
+        """
+        Returns default display name and editor
+        """
+        display_name_setting = self.q(css='.wrapper-comp-setting input[type="text"]:nth-child(2)').attrs('value')[0]
+        editor_setting = self.q(css='.wrapper-comp-setting .input.setting-input :nth-child(1)').text[0]
+        return [display_name_setting, editor_setting]
+
+    @property
+    def keys(self):
+        """
+        Gets setting keys
+        """
+        return self.q(css='.label.setting-label[for]').text
+
+    def set_field_val(self, field_display_name, field_value):
+        """
+        If editing, set the value of a field.
+        """
+        selector = '.xblock-studio_view li.field label:contains("{}") + input'.format(field_display_name)
+        script = "$(arguments[0]).val(arguments[1]).change();"
+        self.browser.execute_script(script, selector, field_value)
+
+    def save(self):
+        """
+        Clicks save button.
+        """
+        click_css(self, '.save-button')
+
+    def _click_button(self, button_name):
+        """
+        Click on a button as specified by `button_name`
+
+        Arguments:
+            button_name (str): button name
+
+        """
+        self.q(css=button_name).first.click()
+        self.wait_for_ajax()
```

Comment:
you are already using click_css() function imported from utils for save button, you can use the same for other button clicks as well instead of writing a new helper